index.html

```diff
@@ -0,0 +1,369 @@
+  <link rel="stylesheet" href="./reset.css">
+  <link rel="stylesheet" href="./styles.css">
+  <link rel="stylesheet" href="./variables.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Alumni+Sans+Pinstripe&family=Bitter&display=swap" rel="stylesheet">    
+  <title>Cofico</title>
+</head>
+<body>
+  <div class="container">
+    <div class="main">
+      <div id="nav">
+        <div class="cofico">
+          <a href="#">
+            <img src="./icon/cofico.svg" alt="cofico">
+          </a>
+        </div>
+
+        <div class="links">
+          <a href="#">
+            Home
+          </a>
+
+          <a href="#">
+            About Us
+          </a>
+
+          <a href="#">
+            Shop
+          </a>
+
+          <a href="#">
+            Sign in
+          </a>
+
+          <button class="button">Sign up</button>
+        </div>
+      </div>
+
+      <h1>The Strong Aroma of
+        <span class="colortext"> Coffee</span>
+          can Instantly Make You <span class="colortext">Feel Better</span>.</h1>
+        <p>Regular black coffee (without milk or cream) is low in calories. In fact, a typical cup of black coffee
+          only contains around 2 calories. However, adding cream or sugar will increase the calorific value.</p>
+
+      <button class="button">Explore More</button>
+    </div>
+
+    <div class="cup-coffee">
+      <img src="./img/cup-of-coffee.png" alt="a cup of coffee">
+    </div>
+
+  </div>
+
+  <div class="container-2">
+    <div class="bags-coffee">
+      <img src="./img/coffee-bags.png" alt="coffee bags">
+    </div>
+
+    <div class="main-2">
+      <h1>The Most Popular Coffee</h1>
+      <p>Regular black coffee (without milk or cream) is low in calories. In fact, a typical cup of black coffee 
+        only contains around 2 calories. However, adding cream or sugar will increase the calorific value.</p>
+      <button class="button">Shop Now</button>
+    </div>
+  </div>
+    <h1 class="heiding">Our Best Services</h1>
+    <p class="text">Regular black coffee (without milk or cream) is low in calories. In fact, a typical cup of black coffee only 
+    contains around 2 calories. However, adding cream or sugar will increase the calorific value.Regular black coffee </p>
+
+  <div class="container-3">
+      <div class="tea">
+        <img src="./icon/tea-leyer.svg" alt="tea leyer">
+        <h2>The Leyer</h2>
+        <p>Regular tea (without milk or cream) is low in calories. In fact, a typical cup of tea</p>
+        <button class="button-2">Explore More</button>
+      </div>
+
+      <div class="chocolate-coffee">
+        <img src="./icon/chocolate-coffee.svg" alt="casual chocolate coffee">
+        <h2>casual chocolate coffee</h2>
+        <p>Regular casual coffee (without milk or cream) is low in calories. In fact, a typical cup of coffee</p>
```

Comment:
move text after tag on a new line if it's long

Reply:
I think that's ok